.github/workflows/APIDigesterCheck.yml

```diff
@@ -0,0 +1,42 @@
+name: Swift API Digester Functionality Check
```

Comment:
We've been using `snake_case` for workflow file names. 
Could you rename this file to `api_digester_check.yml` for consistency?